### ISSUE TITLE
cdpam code missing?

### ISSUE THREAD

**Issue by jonnor** [pranaymanocha/PerceptualAudio#15]
Hi. Thank you for the DPAM model, excellent work and very nice paper!

I am curious to try your CDPAM model. I tried checking out the new cdpam model code.
However the directory seems to be empty in git? In https://github.com/pranaymanocha/PerceptualAudio/tree/master/cdpam
I only see `requirements.txt`, `README.md` and a `sample/` directory?

**Comment by pranaymanocha** [comment]
Hi. Thanks for your comments. The work is under submission, and the idea is to release everything (dataset, trainable model etc.) after we hear back from the reviewers. However, we released the metric in a pip module so that people can try out the metric and see if that works for their task. The pip module supports evaluation the metric stand alone, and training the metric on some downstream task.

**Comment by jonnor** [comment]
Ok, understood. Thank you for the quick response! Looking forward to the paper and the release.

If a preprint is available I would love to read it (my email is in github profile and on my website).

**Comment by calclavia** [comment]
Any specifications on how to use this metric in Pytorch? (Can it be used as a metric during training and what sample rate is expected?)

**Comment by pranaymanocha** [comment]
Yes, it can be used as a metric during training. It expects **sr=22050**. For training, you need to pass tensors (it doesn't automatically change the sampling rate), but for eval you can pass numpy arrays (as shown by cdpam.load_audio() - which also converts automatically into sr=22050 in the backend). If you pass in tensors, then it passes gradients, but if you pass numpy arrays, it doesnt track gradients.

**Comment by jonnor** [comment]
@pranaymanocha Hi. We are working on a paper where we want/need to reference CDPAM. Can you please advise on the best way of doing so? I guess we could reference the Github repo, but if you have an paper or pre-print that would be even nicer!

**Comment by pranaymanocha** [comment]
Hi, If you need a citation immediately, then please cite this github repo. However, if you can wait a few days then (I guess by next week), we expect to be able to provide a more suitable citation (possibly arXiv). Sorry for the confusion!!

**Comment by jonnor** [comment]
Thank you for the reply. Our deadline is still some weeks away, so we will wait for the proper citable artifact and cite that.

**Comment by pranaymanocha** [comment]
@jonnor Hi! I just updated the citation in the cdpam folder. Please have a look. I am also planning to put the paper up on arxiv in the coming 2-3 days, so I'll update you once that's done. Sorry for the trouble earlier.

**Comment by pranaymanocha** [comment]
Hi @jonnor @calclavia I just put the arxiv link of the paper in the repo. Please have a look guys. Thanks!
